salt/modules/bcache.py

```diff
@@ -428,9 +421,7 @@ def config_(dev=None, **kwargs):
         spath = _bcpath(dev)
 
     # filter out 'hidden' kwargs added by our favourite orchestration system
-    updates = dict(
-        [(key, val) for key, val in kwargs.items() if not key.startswith("__")]
-    )
+    updates = {key: val for key, val in kwargs.items() if not key.startswith("__")}
```

Comment:
It's a *teeny* bit faster to do:

```
updates = [(key, val) for key, val in kwargs.items() if not key.startswith("__")]
```, since we get to skip the second `items()` call and head straight to unpacking. Might actually be better to skip this part entirely and just do an `if any(not key.startswith('__'))`. Oh, yeah, that's pretty significant:

```
print(timeit.timeit('''
if any(not key.startswith('__') for key in kwargs):
 for key, val in kwargs.items():
  ...
''', globals={'kwargs': x}))
```

Is about 1s faster than the existing method.